build-deploy-platform-services/Jenkinsfile

```diff
@@ -433,3 +460,36 @@ def loadServiceConfigurationDataModule(buildURL,configLoader,role_arn, region, r
 			error "loadServiceConfigurationDataModule Failed. "+ex.getMessage()
 		}
 }
+
+@NonCPS
+def parseJson(jsonString) {
+    def lazyMap = new groovy.json.JsonSlurperClassic().parseText(jsonString)
+    def m = [:]
+    m.putAll(lazyMap)
+    return m
+}
+
+def loadServiceMetadata(service_id, region){
+	
+	withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+		credentialsId: configLoader.AWS_CREDENTIAL_ID, secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+			
+		def table_name = var_env_name_prefix + "_services_prod"
+		def service_Object = sh (
+				script: "aws --region $region dynamodb get-item --table-name $table_name --key '{\"SERVICE_ID\": {\"S\":\"$service_id\"}}' --output json" ,
+				returnStdout: true
+			).trim()
+		
+		
+		if(service_Object){
+			def service_data = parseJson(service_Object)
+			def data = service_data.Item.SERVICE_METADATA.M
+			def metadata = [:]
+			
+			for(item in data){
+				metadata[item.key] = item.value.S				
+			}			
+			return metadata
+		}
+	}
```

Comment:
region, domain, owner, service name are not part of service_metadata map, so needs to be added from outside.

region comes from configLoader where as the rest come from catalog